pkg/webui/oauth/views/app/index.js

```diff
@@ -78,7 +80,9 @@ export default class OAuthApp extends React.PureComponent {
                     <Route path="/oauth/login" component={Login} />
                     <Route path="/oauth/authorize" component={Authorize} />
                     <Route path="/oauth/register" component={CreateAccount} />
+                    <Route path="/oauth/reset-password" component={ResetPassword} />
                     <Route path="/oauth/code" component={Code} />
+                    <Route path="/oauth/change-password/:user_id/:token" component={ChangePassword}/>
```

Comment:
Correct me if im wrong. Is the `token` param - the old password?

Reply:
no, it's the new token obtained from the reset password and it's used as a parameter to change the password